rustls/src/webpki/client_verifier_builder.rs

```diff
@@ -97,6 +98,17 @@ impl From<CertRevocationListError> for ClientCertVerifierBuilderError {
     }
 }
 
+impl fmt::Display for ClientCertVerifierBuilderError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::NoRootAnchors => write!(f, "no root trust anchors were provided"),
+            Self::InvalidCrl(e) => write!(f, "provided CRL could not be parsed: {:?}", e),
+        }
+    }
+}
+
+impl std::error::Error for ClientCertVerifierBuilderError {}
```

Comment:
small nit: in other places in the codebase I see we tend to import `Error` and alias it to `StdError`:
https://github.com/rustls/rustls/blob/b08776c84d33a522e9f5c70a04ce01f5d5425669/rustls/src/sign.rs#L11

https://github.com/rustls/rustls/blob/b08776c84d33a522e9f5c70a04ce01f5d5425669/rustls/src/sign.rs#L407

Could you match that convention here?

Reply:
Fixed.

Reply:
Thanks! Please squash that into the first commit.

Reply:
Done.

Reply:
Squached to one commit.

Reply:
Thanks